README.md

```
@@ -1,37 +1,79 @@
+| 항목           | 내용                                       |
+| :------------- | :----------------------------------------- |
+| **프로젝트명** | **O+T** (Open The Taste)                   |
+| **주제**       | 데이터 분석 기반 맞춤형 콘텐츠 추천 서비스 |
+| **타겟층**     | 콘텐츠 선택 장애를 겪는 모든 OTT 사용자    |
+| **개발 기간**  | 2025.02.04 ~ 2025.03.20 (약 6주)           |
+
+### 페인포인트
 
-### 콘텐츠 추천
+<p align="center">
+  <img width="45%" src=".github/readme/servey.png" />
```

markdownlint-cli2 (0.21.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)